src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -230,17 +230,9 @@
  *   &lt;property name=&quot;option&quot; value=&quot;bottom&quot;/&gt;
  *   &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
- * </pre>
- * <pre>
- * &lt;?xml version=&quot;1.0&quot;?&gt;
- * &lt;!DOCTYPE suppressions PUBLIC
- *     &quot;-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN&quot;
- *     &quot;https://checkstyle.org/dtds/suppressions_1_2.dtd&quot;&gt;
- *
- * &lt;suppressions&gt;
- *     &lt;!-- message contains no message text to work well in multi-language environments --&gt;
- *     &lt;suppress checks=&quot;ImportOrder&quot; message=&quot;^'java\..*'.*&quot;/&gt;
- * &lt;/suppressions&gt;
+ * &lt;module name="SuppressionXpathSingleFilter"&gt;
+ *   &lt;property name="checks" value="ImportOrder"/&gt;
```

Comment:
> and forget about depending on message that is problematic with multilanguage mode.

I'm confusing about this. So does it mean that I should consider using **xpath** first rather than using **message** because **message** is problematic.

Reply:
I thought about this one more time, there is no way to do this by xpath.
Please simply convert it as is, with `message` .

Reply:
Done.